stories/core_components/GeoPushpinChart.tsx

```diff
@@ -169,4 +169,22 @@ storiesOf("Core components/GeoPushpinChart", module)
                 config,
             }),
         );
+    })
+    .add("with location, size, color, segmentBy, tooltipText with config zoom/center", () => {
```

Comment:
We should remove this case since it's already covered by other cases

Reply:
updated